README.md

```diff
@@ -40,14 +40,25 @@ The TAC is chartered as part of the [Open Source Security Foundation Charter](ht
-| Securing Critical Projects                           | https://github.com/ossf/wg-securing-critical-projects         | [Meeting Notes](https://docs.google.com/document/d/1MIXxadtWsaROpFcJnBtYnQPoyzTCIDhd0IGV8PIV0mQ/edit) | Incubating |
-| Supply Chain Integrity                               | https://github.com/ossf/wg-supply-chain-integrity             | [Meeting Notes](https://docs.google.com/document/d/1xPs2sSbH3I9Ich7OyLOzl85oJshnK8Q6WoAgREE5-zA/edit) | Incubating |
+### Working Groups
+
+| Name                         | Repository                                              | Notes                                                                                                  | Status     |
+| ---------------------------- | ------------------------------------------------------- | ------------------------------------------------------------------------------------------------------ | ---------- |
+| Vulnerability Disclosures    | https://github.com/ossf/wg-vulnerability-disclosures    | [Meeting Notes](https://github.com/ossf/wg-vulnerability-disclosures/tree/main/docs/meeting-notes)     | Incubating |
+| Security Tooling             | https://github.com/ossf/wg-security-tooling             | [Meeting Notes](https://docs.google.com/document/d/1DoB7zgtLsP-JGF77ASkHV7UMofTE2wseniexaa6Q4M8/edit#) | Incubating |
+| Security Best Practices      | https://github.com/ossf/wg-best-practices-os-developers | [Meeting Notes](https://github.com/ossf/wg-best-practices-os-developers/blob/main/meeting-minutes.md)  | Incubating |
+| Identifying Security Threats | https://github.com/ossf/wg-identifying-security-threats | [Meeting Notes](https://docs.google.com/document/d/1AfI0S6VjBCO0ZkULCYZGHuzzW8TPqO3zYxRjzmKvUB4/edit)  | Incubating |
+| Securing Critical Projects   | https://github.com/ossf/wg-securing-critical-projects   | [Meeting Notes](https://docs.google.com/document/d/1MIXxadtWsaROpFcJnBtYnQPoyzTCIDhd0IGV8PIV0mQ/edit)  | Incubating |
+| Supply Chain Integrity       | https://github.com/ossf/wg-supply-chain-integrity       | [Meeting Notes](https://docs.google.com/document/d/1xPs2sSbH3I9Ich7OyLOzl85oJshnK8Q6WoAgREE5-zA/edit)  | Incubating |
+
+
+### Projects
+
+| Name        | Repository          | Notes                                                                                                 | Status     |
+| ----------- | ------------------- | ----------------------------------------------------------------------------------------------------- | ---------- |
+| Sigstore    | github.com/sigstore | [Meeting Notes](https://docs.google.com/document/d/1bsl-Y0KulSD7O_nTekad1sAKOVRb80wyGb-Q5x-zdg0/edit) | Incubating |
+| GTI         | ?                   | ?                                                                                                     | Incubating |
```

Comment:
Let's put "coming soon" for GTI and Project A-O; will be getting public meetings going for these soon. Also can we spell out GTI - GNU Toolchain Infrastructure

Reply:
All set!